src/pages/GoverningBody/Sector/Sector.tsx

```diff
@@ -110,19 +114,31 @@ const Sector = () => {
 
   const getUserAccesses = async () => {
     let user: any = jwt(AuthStore.getToken() as string);
+    let result :any
     await getUserAccess(user.nameid).then(
       response => {
+        result = response
         setUserAccesses(response.data);
       }
     );
+    return result
   }
 
   const getSector = async () => {
     setLoading(true);
     try {
       const response = await getSectorById(+sectorId);
       const sectorViewModel = response.data.sectorViewModel
-      await getUserAccesses();
+      let userAccesses = await getUserAccesses();
+      if(userAccesses.data["ViewAnnouncements"]){
+        const res = (await getAllAnnouncements()).data
+        let shortListedAnnoncements: GoverningBodyAnnouncement[] = [];
+        for(let i = 0; i < res.length && i < 3; i++) {
+          res[i].text = res[i].text.substring(0,40) + (res[i].text.length > 40? "...": "")
```

Comment:
what are the magic numbers 3 and 40? create some constant values to store these numbers